app/src/main/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerViewModel.kt

```diff
@@ -30,6 +30,8 @@ class QuestionPlayerViewModel @Inject constructor() : ObservableViewModel() {
   val isHintBulbVisible = ObservableField(false)
   val isHintOpenedAndUnRevealed = ObservableField(false)
 
+  var isAnswerCorrect = false
+
```

Comment:
remove the extra line in between

Reply:
`line :32`

Reply:
`isAnswerCorrect` is not related to hints so I left a line here.

Reply:
then would it be better if you add it after `line :26`

Reply:
done